1차/윤형진_체육복.py

```diff
@@ -0,0 +1,36 @@
+def solution(n, lost, reserve):
+    answer = 0
+    student_list = [0 for i in range(n)]
+    
+    # 여분 보유 학생에 대해 1로 설정, 없는 학생은 -1, 일반 학생은 0
+    # 여분의 체육복을 빌려주면 1을 0으로 바꿈
+    for i in reserve : 
+        student_list[i - 1] = 1 
+        
+    for i in lost :
+        student_list[i - 1] = student_list[i - 1] - 1 #여분을 가져온 학생도 도난당할 수 있다는 점
+
+    for i in range(n) : 
+        if student_list[i] == -1 :
+            if i == 0 : #가장 앞사람인 경우 보다 앞사람이 존재하지 않음
+                if student_list[i + 1] == 1 :
+                    student_list[i + 1] = 0
+                    student_list[i] = 0
+            elif i == (n - 1) : #가장 뒷사람인 경우 보다 뒷사람이 존재하지 않음
+                if student_list[i - 1] == 1 :
+                    student_list[i - 1] = 0
+                    student_list[i] = 0
+            else :
+                if student_list[i - 1] == 1 :
+                    student_list[i - 1] = 0
+                    student_list[i] = 0
+                elif student_list[i + 1] == 1 :
+                    student_list[i + 1] = 0
+                    student_list[i] = 0
+    
+
+    for i in student_list :
+        if i == 0 or i == 1 :
```

Comment:
```suggestion
        if i >= 0 :
```
- i == 0 or i == 1 으로 처리하는 것보다 i >= 0가 더 가독성 있어 보입니다. 
- 혹은 if i != -1 도 괜찮아 보이네요.
- 이러한 방식이 가능한 이유는 각 학생마다 예비를 1개만, 놓고오는것도 1개만 가능하기 때문입니다.

Reply:
아 그 부분은 생각하지 못했네요 감사합니다